tests/internal/test_packages.py

```diff
@@ -52,6 +52,8 @@ def test_get_distributions():
             importlib_pkgs.add("typing-extensions")
         elif pkg.name == "pkgutil_resolve_name" and "pkgutil-resolve-name" in pkg_resources_ws:
             importlib_pkgs.add("pkgutil-resolve-name")
+        elif pkg.name == "importlib_metadata" and "importlib-metadata" in pkg_resources_ws:
+            importlib_pkgs.add("importlib-metadata")
```

Comment:
Had to add this to address failures like this: https://app.circleci.com/pipelines/github/DataDog/dd-trace-py/62109/workflows/5915fb1c-6dae-4a62-8941-085ffafd9cd1/jobs/3877944